.github/workflows/cd.yml

```
@@ -0,0 +1,38 @@
+name: CD
+
+on:
+  workflow_dispatch:
+  pull_request:
+  push:
+    branches:
+      - main
+  release:
+    types:
+      - published
+
+jobs:
+  dist:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - uses: hynek/build-and-inspect-python-package@v1
```

Looks like this action is at v2 now (the usage example in the README just hasn't been updated). https://github.com/hynek/build-and-inspect-python-package/releases/tag/v2.0.0

Looks like this upgrade also requires updating `actions/download-artifact` to v4 in the `publish` job.

```suggestion
      - uses: hynek/build-and-inspect-python-package@v2
```